.idea/stacks-blockchain.iml

```diff
@@ -10,9 +10,10 @@
       <sourceFolder url="file://$MODULE_DIR$/stx-genesis/src" isTestSource="false" />
       <sourceFolder url="file://$MODULE_DIR$/testnet/puppet-chain/src" isTestSource="false" />
       <sourceFolder url="file://$MODULE_DIR$/testnet/stacks-node/src" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/contrib/tools/relay-server/src" isTestSource="false" />
```

Comment:
i checked in this `.idea` directory :/... @donpdonp was talking about deleting this

can this file at least be dropped for now?

Reply:
update: https://github.com/stacks-network/stacks-blockchain/pull/3523

Reply:
Oh.. well #3523 is open against `develop` right now and this PR is against `next`